processing/src/test/java/org/apache/druid/segment/UnnestColumnValueSelectorCursorTest.java

```diff
@@ -387,44 +384,6 @@ public void test_list_unnest_cursors_user_supplied_list_with_dups()
     Assert.assertEquals(k, 10);
   }
 
-  @Test
```

Comment:
Should these tests be retained, but updated to whatever the new format is? Presumably these addressed some earlier concern, and we should ensure that they still work. Or, are there new tests that cover the same cases that the deleted tests cover?

Reply:
There are new tests that cover these. They are added in the CalciteArraysQueryTest